Source/WebCore/platform/graphics/gstreamer/mse/TrackQueue.cpp

```diff
@@ -48,7 +48,17 @@ void TrackQueue::enqueueObject(GRefPtr<GstMiniObject>&& object)
     ASSERT(isMainThread());
     ASSERT(GST_IS_SAMPLE(object.get()) || GST_IS_EVENT(object.get()));
 
-    GST_TRACE("TrackQueue for '%s': Putting object in the queue: %" GST_PTR_FORMAT ". notEmptyCallback currently %s.", m_trackId.string().utf8().data(), object.get(), m_notEmptyCallback ? "set, will be called" : "unset");
+#ifndef GST_DISABLE_GST_DEBUG
```

Comment:
I don't think a ifdef is needed here, we don't do extra allocations or costly operations here that would be avoided by disabling GST_DEBUG, afaik.